frontend/components/root.jsx

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import { HashRouter } from 'react-router-dom';
 
 import { library } from '@fortawesome/fontawesome-svg-core';
-import { faLock, faUnlock, faArrowLeft, faPoo, faGlassCheers, faAngry, faHorse, faAnchor, faCarrot, faCloudRain, faGlasses, faSeedling, faChessPawn, faCookieBite, faGasPump, faInfinity, faGhost, faDiceFive, faGem, faPuzzlePiece, faToilet, faPencilAlt, faMoon } from '@fortawesome/free-solid-svg-icons';
+import { faLock, faUnlock, faArrowLeft, faArrowRight, faArrowUp, faArrowDown, faPoo, faGlassCheers, faAngry, faHorse, faAnchor, faCarrot, faCloudRain, faGlasses, faSeedling, faChessPawn, faCookieBite, faGasPump, faInfinity, faGhost, faDiceFive, faGem, faPuzzlePiece, faToilet, faPencilAlt, faMoon, faStar } from '@fortawesome/free-solid-svg-icons';
```

Comment:
if I wanted to use other fontawesome icons, do I have to import them here and then add to the library below in order to use them in a component later on?

Reply:
i believe so. i haven't been able to get them to work otherwise? seems cumbersome, might try to look into a way to not have to do this